telegram/_chat.py

```diff
@@ -913,6 +905,16 @@ def __init__(
         self.business_location: Optional["BusinessLocation"] = business_location
         self.business_opening_hours: Optional["BusinessOpeningHours"] = business_opening_hours
 
+        if self.__class__.__name__ == "Chat":
```

Comment:
`self.__class__ is Chat` does not work due to recursiveness, I guess?

Reply:
that wasn't my first instinct, but that works as well so doing that.